### ISSUE TITLE
Unable to move focus to add-in elements when hitting Tab

### ISSUE THREAD

**Issue by stzva** [OfficeDev/office-js#1196]
I'd like to report the following ***accessibility*** issue in the Outlook add-in. I am not able to move focus from the 'pin' and 'close' buttons to the focus-able elements inside the add-in by hitting Tab on the keyboard.

## Expected Behavior
1) I expect to be able to move the focus from the 'pin' and 'close' buttons to the focus-able elements inside the add-in by hitting Tab on the keyboard. 
2) Shift+Tab should move the focus backwards to the 'close' and 'pin' buttons. 
3) Focus should be trapped inside the add-in container while it is open.

## Current Behavior

When an Outlook add-in first loads and the user hits Tab, the focus is placed on the close button in the header. A second Tab moves the focus (outline is visible) to a div wrapping the add-in iframe. Subsequent Tab hits seem to break the focus order instead of moving the focus to the first focus-able element in the iframe document.

I can see the general structure of the add-in container as follows:

```
[Addin name]  [Pin icon] [X icon]
<div tabindex="0" role="complementary" aria-label="Add-in name. Press tab to enter the add-in">
    <div tabindex="0">
       <iframe>
         .....
       </iframe>
     </div>
</div>
```

With the NVDA screen reader enabled, the reader announces "Close add-in pane  button"  on the first Tab stop and "Add-in name. Press tab to enter the add-in.  complementary landmark" on the second Tab stop. The reader announces nothing on subsequent Tab's.

## Steps to Reproduce
1. Select an email.
2. Right-click on it and click to open one of the available add-ins.
3. When the add-in is open, hit Tab to focus the 'close' button.
4. Hit Tab two or three times more.

## Context
My company is trying to build an accessible add-in, conforming to the WCAG 2.0 standards.

## Your Environment
* Platform: PC desktop
* Host: Office 355, Outlook
* Operating System: Windows 10, 64-bit
* Browsers: 
Chrome Version 83.0.4103.61
Firefox Version 76.0.1
Microsoft Edge Version 83.0.478.44
Internet Explorer 11 Version 11.836.18362.0

**Comment by exextoc** [comment]
Can you please specify, this is happening only for Outlook web or Win32 client as well?

**Comment by stzva** [comment]
> Can you please specify, this is happening only for Outlook web or Win32 client as well?

The described issue is happening only for Outlook web.

**Comment by exextoc** [comment]
We can reproduce the issue. Thanks for reporting this issue regarding move focus when hitting the tab. It has been put on our backlog. We unfortunately have no timelines to share at this point.

**Comment by exextoc** [comment]
We have fixed this issue and it will take ~1 week for 100% rollout.
